.github/data/dependencies.yml

```diff
@@ -69,6 +69,8 @@
   requires: python
   stacks:
     - id: io.buildpacks.stacks.bionic
```

Comment:
The .NET Core dependencies that have a similar set up have `*` listed as the `version-constraint` for Bionic. Can you add that here?

Reply:
done